php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -2544,17 +2550,7 @@ abstract class NDB_BVL_Instrument extends NDB_Page
      */
     function _toJSONParsePage(): array
     {
-
-        if (get_class($this->form) === 'LorisForm') {
-            $formArray = $this->form->toElementArray();
-        } else {
-            $smarty   = new Smarty_neurodb;
-            $renderer = new HTML_QuickForm_Renderer_Array($smarty);
-
-            $this->form->accept($renderer);
-
-            $formArray = $renderer->toArray();
-        }
+        $formArray = $this->form->toElementArray();
```

Comment:
What about the other part of the else statement? What is this trying to solve? I think this change of behaviour is likely to break things with different types of instruments

Reply:
I believe there was an issue with `HTML_QuickForm_Renderer_Array` because it doesn't exist in the code. 

I made this PR 6 months ago so I'm not sure about the details.

Reply:
I just did a little investigation of the impact of losing this else case and I think it means the API would be broken for the endpoint which gets the instrument in a JSON format for projects that had old instruments that couldn't be updated to LorisForm and were still using QuickForm.

There was only 1 such instrument that I could find when I searched the IBIS repo, and I'm not aware of anyone/thing depending on that endpoint of the API, so I guess it should be fine.

Reply:
@xlecours or @ridz1208 do you want to chime in or know of anything using the `/projects/$proj/instruments/$instrumentname` endpoint?

Reply:
We can add a note in the CHANGELOG about this if it ends up being a breaking change.